svelte-development/src/MapPoints.svelte

```diff
@@ -1,29 +1,31 @@
 <script>
   import { spring } from "svelte/motion";
+  import { getContext } from 'svelte';
   import { scaleLinear, extent, min, max, select, selectAll, groups, line } from 'd3';
-  import { forceSimulation, forceCollide, forceLink, forceX, forceY } from "d3-force"
-
+  import { forceSimulation, forceCollide, forceLink, forceX, forceY } from "d3-force";
```

Comment:
@Yuriko-Schumacher I should have commented on this on your last PR, but I don't understand the reason for using a force-directed graph here. I feel like that's computationally expensive, especially because we know where the points are going to go. I also complicates drawing a line between two points.